README.md

```diff
@@ -195,10 +195,17 @@ Add the following to your `config.php`:
                     2 => 'dbuser',
                     3 => 'dbuserpassword',
                     4 => 'xmppdomain',
+		    5 => true,
```

Comment:
the indention should be the same here I guess

Reply:
Fixed.